src/styles/product_card.scss

```diff
@@ -0,0 +1,112 @@
+.product {
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
```

Comment:
The `transform: scale(1.2);` rule can cause the element to overflow its parent, if the parent's size is not flexible. Make sure this is the desired behavior. If not, consider adding `overflow: hidden;` to the parent element.